aptos-move/framework/supra-framework/sources/configs/supra_config.spec.move

```diff
@@ -0,0 +1,53 @@
+spec supra_framework::supra_config {
+    /// <high-level-req>
+    /// No.: 1
+    /// Requirement: During genesis, the Supra framework account should be assigned the supra config resource.
+    /// Criticality: Medium
+    /// Implementation: The supra_config::initialize function calls the assert_supra_framework function to ensure
+    /// that the signer is the supra_framework and then assigns the SupraConfig resource to it.
+    /// Enforcement: Formally verified via [high-level-req-1](initialize).
+    ///
+    /// No.: 2
+    /// Requirement: Only aptos framework account is allowed to update the supra protocol configuration.
```

Comment:
```suggestion
/// Requirement: Only supra framework account is allowed to update the supra protocol configuration.
```
Should this be Supra framework?

Reply:
Yes. I didn't update the docs very carefully. We'll have to update that in future PR.